packages/react-component-library/.storybook/preview.js

```diff
@@ -58,8 +59,10 @@ export const parameters = {
 
 export const decorators = [
   (Story) => (
-    <GlobalStyleProvider>
-      <Story />
-    </GlobalStyleProvider>
+    <React.StrictMode>
```

Comment:
Does the strictMode option mentioned here work? https://storybook.js.org/blog/storybook-6-1/

(BTW I noticed that these decorators runs multiple times on the Docs tab, which is causing some duplicate styles from `GlobalStyleProvider`.)

Reply:
Hmm i'll investigate I was under the impression each story was nested within an iframe - I guess not within docs.

Reply:
Doing it the using the option I get a lot of "Rendered more hooks than during the previous render." messages which is due to https://github.com/storybookjs/storybook/issues/15223 it seems...

So it may be this way is better for now.

Reply:
Created ticket to look further into enforcing singleton behaviour of provider: https://github.com/defencedigital/mod-uk-design-system/issues/3259